platform/srf06-cc26xx/contiki-main.c

```diff
@@ -123,6 +126,10 @@ set_rf_params(void)
     printf("%02x\n", linkaddr_node_addr.u8[i]);
   }
 #endif
+
+  /* also set the global node id */
+  node_id = short_addr;
```

Comment:
I cannot immediately see why we need this. Is this required by TSCH?

The CPU makefile does this:
```
ifdef NODEID
  CFLAGS += -DIEEE_ADDR_NODE_ID=$(NODEID)
endif
```

Is this not suitable?